odxtools/cli/find.py

```diff
@@ -120,6 +121,14 @@ def add_subparser(subparsers: "argparse._SubParsersAction") -> None:
         help="Relax output formatting rules (allow unknown bitlengths for ascii representation)",
     )
 
+    parser.add_argument(
+        "-od",
+        "--object-details",
+        action="store_true",
+        required=False,
+        help="Print all object details instead of just the name",
```

Comment:
hm, I do not really like the name of this parameter because, if I understood this correctly, there are more attributes considered in the printed tables?  maybe rename it to `--print-plumbing` (in reference to [git](https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain))?

Reply:
A table with selected attributes is only displayed for request and response parameters. The global negative responses and DOPs are displayed by their attribute `short_name`. 
I renamed the argument to `--plumbing-output`. When specified, the full object is printed in addition to the table (for request and response parameters) or instead of the name (for global negative responses and DOPs).